test/test_api/test_api.py

```diff
@@ -308,6 +309,8 @@ def test_goto_follow_builtin_imports(Script):
 
 def test_fuzzy_completion(Script):
     script = Script('string =  "hello"\nstring.upper')
-    assert ['isupper', 'upper'] == [comp.name 
-                                    for comp in 
-                                    script.completions(fuzzy=True)]
+    assert ['isupper',
+            'upper'] == [comp.name for comp in script.completions(fuzzy=True)]
+    script = Script('import math\nmath.og')
+    assert ['copysign', 'log', 'log10', 'log1p',
+            'log2'] == [comp.name for comp in script.completions(fuzzy=True)]
```

Comment:
Please undo all of the changes except the last two lines in this commit: 48ffc5473abc3f6828fa5afe3d0859ad7e6e4d8e

I don't like this reformatting, it's not how I usually format my code (character limit in Jedi is 99 and I like parentheses on newlines if they were there already ;-))

However the test looks really good, thanks for that!